practice-one/src/pages/Products/components/FilterOptions/Categories.tsx

```diff
@@ -0,0 +1,50 @@
+import { TCategory } from '../../../../types'
+import Button from '../../../../components/Button'
+
+const Categories = ({
+  value,
+  setValue,
+}: {
+  value?: TCategory
+  setValue: React.Dispatch<React.SetStateAction<TCategory | undefined>>
+}) => {
+  return (
+    <div className="border-b-2 border-solid py-4">
+      <p className=" mb-2 text-lg capitalize">categories</p>
+      <div className="flex gap-2">
+        <Button
+          size="small"
+          checked={value === 'men'}
```

Comment:
create enum for these categories please

Reply:
update for size and color please

Reply:
The type file is already in the branch `feature/implement-products-page`

![image](https://github.com/HoangLiarPhuongLe/react-training/assets/110210274/819186a5-cc56-470f-8cd4-0ae0a5758ff4)